source/run-agg-pipelines.txt

```diff
@@ -140,69 +181,45 @@ First Stage
      The ``$project`` pipeline in the example suppresses the ``_id``
      field from the documents it passes to the next stage.
 
+  The ``$project`` pipeline in the example transforms the example
```

Comment:
I think "in the example" muddies this a bit. I think we can get rid of it.

```suggestion
  The ``$project`` stage transforms the example
```

Reply:
updated

Reply:
updated